src/20250426_Multi_MayFundingUpdate/MayFundingUpdate.md

```diff
@@ -0,0 +1,135 @@
+
+## Specification
+
+This proposal shall be submitted as several AIPs that align with operational readiness timelines and ease of review given the volume of funds being moved.
+
+### Bridge Assets to Ethereum
+
+Bridge the following assets to Ethereum.
+
+|                                       Polygon                                       |                                    Arbitrum                                    |                                          Optimism                                          |
+| :---------------------------------------------------------------------------------: | :----------------------------------------------------------------------------: | :----------------------------------------------------------------------------------------: |
+|   [DAI](https://polygonscan.com/token/0x8f3cf7ad23cd3cadbd9735aff958023239c6a063)   |  [DAI](https://arbiscan.io/token/0xda10009cbd5d07dd0cecc66161fc93d7c9000da1)   |  [DAI](https://optimistic.etherscan.io/token/0xda10009cbd5d07dd0cecc66161fc93d7c9000da1)   |
+|  [USDT](https://polygonscan.com/token/0xc2132d05d31c914a87c6611c10748aeb04b58e8f)   | [USDC.e](https://arbiscan.io/token/0xff970a61a04b1ca14834a43f5de4533ebddb5cc8) |  [USDT](https://optimistic.etherscan.io/token/0x94b008aa00579c1307b0ef2c499ad98a8ce58e58)  |
+| [ USDC.e](https://polygonscan.com/token/0x2791bca1f2de4661ed88a30c99a7a9449aa84174) |                                                                                | [USDC.e](https://optimistic.etherscan.io/token/0x7f5c764cbc14f9669b88837ca1490cca17c31607) |
+|  [WPOL](https://polygonscan.com/token/0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270)   |                                                                                |
+|  [WBTC](https://polygonscan.com/token/0x1bfd67037b42cf73acf2047067bd4f2c47d9bfd6)   |                                                                                |
+| [MATICX](https://polygonscan.com/token/0xfa68fb4628dff1028cfec22b4162fccd0d45efb6)  |                                                                                |
+|   [ETH](https://polygonscan.com/token/0xe50fa9b3c56ffb159cb0fca61f5c9d750e8128c8)   |
+
+### Ethereum - Swaps
+
+Swap the following from the Collector as outlined in the table below:
+
+| Ethereum | Amount | Swap |
+| :------: | :----: | :--: |
+|   USDT   | 3.00M  | GHO  |
+|   USDC   | 3.00M  | GHO  |
+|   DAI    |  All   | USDS |
+|   LUSD   |  All   | WETH |
+|   CVX    |  All   | WETH |
+|   FRAX   |  All   | WETH |
+
+### Create Allowances
+
+**Acquire WETH**
+
+Create a MATIC and stMATIC allowance for the Aave Finance Committee to then swap each asset to ETH and transfer to the Treasury.
+
+- Asset: `MATIC` [0x7D1AfA7B718fb893dB30A3aBc0Cfc608AaCfeBB0](https://etherscan.io/address/0x7d1afa7b718fb893db30a3abc0cfc608aacfebb0)
+- Amount: ~580,156
```

Comment:
I would specify that the allowance is for the max amount, which currently is x. as in the code is full balance and not this specific amount